src/views/record-browser/table-data/table-data.component.js

```diff
@@ -67,9 +68,35 @@
 		}
 
 		function getRows(columns, data) {
-
 			// Convert document objects into arrays with the same order as `columns`
 			return data.map(document => columns.map(column => document[column.name]));
 		}
+
+		// Modal that pops up when clicking on delete row icon
+		$scope.showConfirm = function(row) {
+			var result = modalHelper.confirmDelete('record');
+			var row_id = row[0];
+			result.then(function() {
+				// deletes row from database but not the view
+				api.document.remove(vm.schemaName, row_id)
+				// deletes row from view but not the database
+					.then(function (response) {
+						var row_idIndex = vm.rows.indexOf(row);
+						if (row_idIndex > -1) {
+							vm.rows.splice(row_idIndex, 1);
+						}
+						// see src/services/toast.service.js
```

Comment:
This is implied. Furthermore, the path may change and make this comment invalid.